site/docs/master/api-types/volumesnapshotlocation.md

```diff
@@ -40,7 +40,7 @@ The configurable parameters are as follows:
 
 | Key | Type | Default | Meaning |
 | --- | --- | --- | --- |
-| `region` | string | Empty | *Example*: "us-east-1"<br><br>See [AWS documentation][3] for the full list.<br><br>Queried from the AWS S3 API if not provided. |
+| `region` | string | Empty | *Example*: "us-east-1"<br><br>See [AWS documentation][3] for the full list.<br><br>You have to specify it. It is not queried from the AWS S3 API if not provided. |
```

Comment:
Let's remove the sentence about it not being queried from the S3 API, since S3 is not relevant here (this is for EBS snapshots). You could just say "Required field."